ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/LondonTargetingGasLimitCalculator.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.mainnet;
+
+import org.hyperledger.besu.ethereum.mainnet.feemarket.LondonFeeMarket;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class LondonTargetingGasLimitCalculator extends FrontierTargetingGasLimitCalculator {
+  private static final Logger LOG = LogManager.getLogger();
+  private final long londonForkBlock;
+
+  public LondonTargetingGasLimitCalculator(final long londonForkBlock) {
+    this(1024L, 5000L, Long.MAX_VALUE, londonForkBlock);
```

Comment:
why not using DEFAULT_MIN_GAS_LIMIT and DEFAULT_MAX_CONSTANT_ADMUSTMENT_INCREMENT here ?

Reply:
Ah, I added the constants after I added the London gas calculator.  :+1: